galera/meta/config.yml

```diff
@@ -0,0 +1,12 @@
+config:
+  {%- if pillar.galera.master is defined %}
```

Comment:
The problem is that I need to define at least master role. Otherwise it would not work without formula modification.